en/ros/ros2_comm.md

```diff
@@ -448,19 +449,14 @@ No further action is required by the user.
 #### ROS2 nodes use the Gazebo clock as time source
 
 In this scenario, ROS2 also uses the Gazebo `/clock` topic as time source.
-This approach makes sense if the Gazebo simulation is running with real time factor different from one or if ROS2 need to directly interact with Gazebo.
-The procedure requires to:
+This approach makes sense if the Gazebo simulation is running with real time factor different from one, or if ROS2 needs to directly interact with Gazebo.
+The procedure requires you to:
 
 - Directly bridge ROS and Gazebo using [ros_gz](https://github.com/gazebosim/ros_gz) and the [ros_gz_bridge](https://github.com/gazebosim/ros_gz) package.
-- Bridge the Gazebo `/clock` topic over ROS
+- Bridge the Gazebo `/clock` topic over ROS:
+
   ```sh
   ros2 run ros_gz_bridge parameter_bridge /clock@rosgraph_msgs/msg/Clock[gz.msgs.Clock
-  ```
-- Run all your nodes with the parameter `use_sim_time` set to `true` (see [ROS clock and Time design](https://design.ros2.org/articles/clock_and_time.html)).
-
-On the PX4 side, you are only required to stop the uXRCE-DDS time synchronization, setting the parameter [UXRCE_DDS_SYNCT](../advanced_config/parameter_reference.md#UXRCE_DDS_SYNCT) to `false`.
-
-By doing so, Gazebo will act as main and only time source for both ROS2 and PX4.
```

Comment:
@hamishwillee , why this was removed?

Reply:
Must have been a selection error when I edited through github. Sorry, can you add it back?

Reply:
Of course!

Reply:
Thanks. I'd to it, but I've already screwed up once!